crates/bevy_pbr/src/atmosphere/resources.rs

```diff
@@ -40,18 +45,25 @@ pub(crate) struct RenderSkyBindGroupLayouts {
     pub fragment_shader: Handle<Shader>,
 }
 
+const FILTERED_TEX: TextureSampleType = TextureSampleType::Float { filterable: true };
```

Comment:
This seems like a bit of an antipattern.. to shorten code. why not just have a `TextureSampleType:FilterableFloat` ?